subsystem/hcp/default_ingress_resource_test.go

```diff
@@ -0,0 +1,161 @@
+/*
+Copyright (c) 2024 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package hcp
+
+import (
+	"net/http"
+
+	. "github.com/onsi/ginkgo/v2"                      // nolint
+	. "github.com/onsi/gomega"                         // nolint
+	. "github.com/onsi/gomega/ghttp"                   // nolint
+	. "github.com/openshift-online/ocm-sdk-go/testing" // nolint
+)
+
+var _ = Describe("rhcs_cluster_rosa_hcp - default ingress", func() {
+
+	defaultDay1Template := `{
+						 "kind": "IngressList",
+						 "href": "/api/clusters_mgmt/v1/clusters/123/ingresses",
+						 "page": 1,
+						 "size": 1,
+						 "total": 1,
+						 "items": [
+						   {
+							 "kind": "Ingress",
+							 "href": "/api/clusters_mgmt/v1/clusters/123/ingresses/d6z2",
+							 "id": "d6z2",
+							 "listening": "external",
+							 "default": true,
+							 "dns_name": "redhat.com"
+						   }
+						 ]
+						}
+						`
+	clusterReady := `
+						{
+							"kind": "Cluster",
+							"id": "123",
+							"href": "/api/clusters_mgmt/v1/clusters/123",
+							"name": "cluster",
+							"state": "ready"
+						}
+					`
+
+	It("Updates ListerningMethod", func() {
```

Comment:
```suggestion
	It("Updates ListeningMethod", func() {
```

Reply:
@gdbranco thanks for pointing this typo.
Can you help to take a look again? Thanks